e2e-tests/production-runtime/package.json

```diff
@@ -30,7 +30,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "cross-env CYPRESS_SUPPORT=y gatsby build",
+    "build": "cross-env FROM_COMMAND_LINE=YES CYPRESS_SUPPORT=y gatsby build",
```

Comment:
Consider adding env var with `GATSBY_` prefix to make sure it's available both for browser bundle and for getServerData